test/validation/validate_kinetic.cpp

```diff
@@ -38,16 +39,12 @@ void run_kinetic_dt(
     probe_label plabels[1] = {{"soma.mid", {0u, 0u}}};
 
     meta["sim"] = "arbor";
-    meta["backend_kind"] = to_string(backend);
+    meta["backend_kind"] = arb::has_gpu(context)? "gpu": "multicore";
```

Comment:
So are there only two `backend_kind`s now? I see the enum still exists somewhere, and the name comes up in these external interfaces, but the mapping has been changed.